package com.exception;

import java.util.ArrayList;

import java.util.List;

import java.util.Scanner;

public class ExceptionPractice {

	public static void main(String[] args) {

		//  Arithmetic Exception 
		
		// + - / * %
		
//		int a = 10;
//		
//		int b = 0;
//		
//		System.out.println(a/b);
		
		//  Null Pointer Exception
		
//		String s = null;
//		
//		int le = s.length();
//		
//		System.out.println(le);
		
		// Input MisMatch Exception
		
//		 Scanner sc = new Scanner(System.in);
//		   
//		 System.out.println("Enter The Number:- ");
//		   
//		 int num = sc.nextInt();
//		   
//		 System.out.println("The Number Is:- " + num);
		
		
		//   Array Index Out Of Bound Exception

//  		int a[] = new int[5];
//		
//		a[0] = 10;
//		
//		a[1] = 23;
//		
//		a[2] = 30;
//		
//		a[3] = 60;
//		
//		a[4] = 67;
//		
//		a[5] = 77;
//        
//			for (int i = 0; i < a.length; i++) { 
//				
//			     System.out.println(a[i]);
//				
//			}	
		
		// String Index Out Of Bound Exception
		
//		String s = "Aravind";
//		
//		char ca = s.charAt(7);
//		
//		System.out.println(ca);
		
		// Index Out Of Bound Exception
		
		//3)  get()
		
		//array list

      //  List<Object> s1 = new ArrayList<Object>();
        
        // 1) add
        
//        s1.add("Jagadesh");
//        
//        s1.add(23);
//        
//        s1.add(34.5);
//        
//        s1.add('S');
//        
//        s1.add(7);
//		 
//		 Object ob = s1.get(5);
//		 
//		 System.out.println(ob);
        
        
//		String s = "1234eeeew";
//		
//		int pi = Integer.parseInt(s);
//		
//		System.out.println(pi);
		
	
		
		
		
		
		
		
	}

}
